### ISSUE TITLE
Tabs for Multiple Files

### ISSUE THREAD

**Issue by infosisio** [lauthieb/code-notes#72]
Right now if you add multiple files to a snippet, it just shows the one after the other.

Why not do it elegantly with tabs?

**Comment by lauthieb** [comment]
Great idea!
I don't have time to make this live but I would love to help you if you want to make a PR ;)
Regards

**Comment by Andrei0872** [comment]
Is anybody working on this right now ?

**Comment by infosisio** [comment]
@Andrei0872 No, I don't think so. Are you interested in doing it? It would be great.

**Comment by Andrei0872** [comment]
@infosisio it would be my pleasure. I'll start working on it, then. Thanks!
